crates/hera-exex/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+mod exex;
```

Comment:
Can we add the top-level module attributes here?

```suggestion
#![doc(issue_tracker_base_url = "https://github.com/paradigmxyz/op-rs/issues/")]
#![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
#![cfg_attr(not(test), warn(unused_crate_dependencies))]
```